ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!---
+Hi! This template is here to help us collect optimal issue reports. Feel free to adapt it to your situation.
+First, check whether the issue has already been reported. If so, please add this report to that issue thread. If not, proceed to file this new issue.
+Next, please provide a one to four sentence summary of the issue. If you do your very best to be clear, concise and accurate here, other Rustaceans will be eager to assist. Also try to mention which GitHub labels this issue likely needs and/or what parts of Rust it relates to.
+--->
+
+## Code sample
+<!---
+Please craft a minimal/simple and self-contained code sample that (re)produces the issue. If you create a formal test case straight away, that'll be even more helpful.
+--->
+`main.rs`:
+```rust
+
+```
+
+## Expected behavior
+<!---
+Please concisely explain what you expected instead of encountering of the issue. Check if it's helpful to refer to some relevant externalities.
+* Could you refer to parts of official documentation, standards, comments in this codebase, etc.?
+* Did/does the issue not occur with past/other versions?
+* In case it's a correctness issue, then what's the relevant behavior of other implementations (for other languages)?.
+* Are there very similar (perhaps informal) issue reports outside this issue tracker?
+--->
+
+## Actual behavior
+<!---
+Please paste here any human readable output in full. Important information can be in the initial details, so don't cut.
+--->
+
+## Meta
+`rustc --version --verbose`:
+```
+
+```
+
+## Backtrace
+<!---
+Finally, if you haven't shared it yet, reproduce the issue with your minimal example code and share the backtrace you obtained.
```

Comment:
> haven't shared it yet

What’s the “it” here?

This also should mention, that the section is optional and only relevant for compiler crashes.